accord-core/src/main/java/accord/impl/CoordinateDurabilityScheduling.java

```diff
@@ -214,10 +215,15 @@ private void run()
     private void startShardSync(Ranges ranges)
     {
         TxnId at = node.nextTxnId(ExclusiveSyncPoint, Domain.Range);
-        node.scheduler().once(() -> node.withEpoch(at.epoch(), () -> {
+        node.scheduler().once(() -> node.withEpoch(at.epoch(), (ignored, withEpochFailure) -> {
+                           if (withEpochFailure != null)
+                           {
+                               logger.trace("Exception waiting for epoch before coordinating exclusive sync point for local shard durability, epoch {}: {}", at.epoch(), getStackTraceAsString(withEpochFailure));
+                               return;
+                           }
                            CoordinateSyncPoint.exclusive(node, at, ranges)
                                .addCallback((success, fail) -> {
-                                   if (fail != null) logger.trace("Exception coordinating exclusive sync point for local shard durability of {}", ranges, fail);
+                                   if (fail != null) logger.trace("Exception coordinating exclusive sync point for local shard durability of {}: {}", ranges, getStackTraceAsString(fail));
```

Comment:
why are you doing `getStackTraceAsString`?  logger does this for you and cheaper (as it can be only when trace is enabled)

Reply:
You can't do string formatting and provide exceptions at the same time. It's the failure case so I could do the string formatting every time. The way this worked before the exception doesn't appear at all.

Reply:
> You can't do string formatting and provide exceptions at the same Tim

yes you can...

Reply:
```
logger.info("What {} {}", "is", "happening", new Throwable());
```. 

produces 

```
INFO  [main] 2024-07-02 13:52:56,336 EpochGossipTest.java:95 - What is happening
java.lang.Throwable: null
	at org.apache.cassandra.service.accord.EpochGossipTest.main(EpochGossipTest.java:95)
```

Reply:
I'll put it back then!